### DISCUSSION TITLE
[委托]悬赏一好用的追番软件！

### DISCUSSION THREAD

**Discussion by mmango-bit** [share121/inter-knot#3427] [其他]
众所周知，O站被莫名其妙爆破了，囧次元又不太好用，现悬赏一好用的追番软件，用于观看我可爱的老八。
悬赏金额：400000000丁尼
![1716214247799](https://github.com/user-attachments/assets/c3395f8e-1128-4f9b-81cc-891c8e08d029)

**Comment by lllliufeng** [comment]
Kazumi（
Github上就能搜到

**Reply by mmango-bit** [reply]
感谢！刚下，感觉应该很好用。委托的奖励我会通过米忽悠给你哒！

**Comment by 15666851105** [comment]
cillcill怎么样？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by mmango-bit** [reply]
找不到安装包。。。。

**Reply by 15666851105** [reply]
浏览器上可以搜到，下载后进入软件它会提示你更新，这样就可以用到正常使用的cillcill了
也是有弹幕的，签到送VIP
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3d1687e289addc7611eba2c8585f4a9f_5782304423493346054.png" alt="丽都漫步-安比" width="75" >

**Reply by mmango-bit** [reply]
我只找到一个clicli，网页版的，没得APP，不过这网页做得貌似还不错

**Comment by FaeCircuit** [comment]
推荐巴哈姆特，但是需要科学上网到台湾
